src/content/ja/2021/resource-hints.md

```diff
@@ -0,0 +1,648 @@
+      </td>
+      <td class="numeric">1.2%</td>
+      <td class="numeric">3.9%</td>
+      <td class="numeric">0.2%</td>
+      <td class="numeric">5.3%</td>
+    </tr>
+    <tr>
+      <td>
+          www.googletagmanager.com
+      </td>
+      <td class="numeric">1.9%</td>
+      <td class="numeric">2.7%</td>
+      <td class="numeric">0.2%</td>
+      <td class="numeric">4.8%</td>
+    </tr>
+    <tr>
+      <td>static.parastorage.com</td>
+      <td class="numeric">-</td>
+      <td class="numeric">-</td>
+      <td class="numeric">4.7%</td>
+      <td class="numeric">4.7%</td>
+    </tr>
+    <tr>
+      <td>ajax.googleapis.com</td>
+      <td class="numeric">2.2%</td>
+      <td class="numeric">1.6%</td>
+      <td class="numeric">0.3%</td>
+      <td class="numeric">4.1%</td>
+    </tr>
+    <tr>
+      <td>www.google.com</td>
+      <td class="numeric">2.7%</td>
+      <td class="numeric">1.0%</td>
+      <td class="numeric">0.1%</td>
+      <td class="numeric">3.8%</td>
+    </tr>
+    <tr>
+      <td>images.squarespace-cdn.com</td>
+      <td class="numeric">-</td>
+      <td class="numeric">3.5%</td>
+      <td class="numeric">-</td>
+      <td class="numeric">3.5%</td>
+    </tr>
+    <tr>
+      <td>cdnjs.cloudflare.com</td>
+      <td class="numeric">1.6%</td>
+      <td class="numeric">1.0%</td>
+      <td class="numeric">0.4%</td>
+      <td class="numeric">2.9%</td>
+    </tr>
+    <tr>
+      <td>monorail-edge.shopifysvc.com</td>
+      <td class="numeric">2.0%</td>
+      <td class="numeric">0.8%</td>
+      <td class="numeric">-</td>
+      <td class="numeric">2.8%</td>
+    </tr>
+    <tr>
+      <td>fonts.shopifycdn.com</td>
+      <td class="numeric">-</td>
+      <td class="numeric">1.1%</td>
+      <td class="numeric">1.0%</td>
+      <td class="numeric">2.1%</td>
+    </tr>
+  </tbody>
+</table>
+
+
+<figcaption>{{ figure_link(caption="モバイル端末でリソースヒントを使用しているもっとも普及しているサードパーティ接続。", sheets_gid="1254656281", sql_file="preload_host_by_url.sql") }}</figcaption>
+</figure>
+
+上の表を分析すると、`preload` ヒントを含む全ページの36.7%がadservice.google.comでホストされているリソースをプリロードしています。s.w.orgホストは `dns-prefetch` でもっとも普及しているドメインで、WordPressサイト（バージョン4.6以降）で、ブラウザがネイティブの絵文字をサポートしていないと検出された場合にTwemoji CDNからのSVG画像を読み込むために使用されています。Google Fonts関連のサービスでは、`fonts.gstatic.com` と `fonts.googleapis.com` が `preconnect` ディレクティブを使用するホストとしてもっとも普及しているようです。
+
+{{ figure_markup(
+  image="google-fonts.png",
+  alt="Google Fontsのフォントを埋め込むための手順",
+  caption='Google Fontsの説明では、fonts.gstatic.comとfonts.googleapis.comへ事前接続するようになっています。（出典: <a hreflang="en" href="https://fonts.google.com/">Google Fonts</a>）',
+  description='Code snippet titled "Use on the web" and subtitled with "To embed a font, copy the code into the `<head>` of your html". Code snippet states `<link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">`',
```

Comment:
Should we translate bits of this description?

Reply:
Thanks, I translated